lib/reporting/cloudwatch_client.rb

```diff
@@ -27,7 +32,8 @@ def initialize(
       slice_interval: 1.day,
       logger: nil,
       progress: true,
-      log_group_name: 'prod_/srv/idp/shared/log/events.log'
+      log_group_name: 'prod_/srv/idp/shared/log/events.log',
+      count_distinct: nil
```

Comment:
IMO this option should live outside the `cloudwatch_client`

Reply:
Reset all changes to this file in 5fcad87 and bbe682b.